relay-dynamic-config/src/defaults.rs

```diff
@@ -24,7 +24,7 @@ const MOBILE_OPS: &[&str] = &[
 const APP_START_ROOT_SPAN_DESCRIPTIONS: &[&str] = &["Cold Start", "Warm Start"];
 
 /// A list of patterns found in MongoDB queries.
-const MONGODB_QUERIES: &[&str] = &["*\"$*", "{*", "*({*", "*[{*"];
+const MONGODB_QUERIES: &[&str] = &["*\"$*", r"\{*", r"*(\{*", r"*\[\{*"];
```

Comment:
Is the problem that we just needed to escape special characters?

Reply:
Yes, added an example failure to the description.